static/quick-start/linux.sh

```diff
@@ -239,8 +245,8 @@ do_install_demo() {
 	echo
 	set -x
 
-  $sh_c "curl -L -o dev.tar.gz https://github.com/ioFog/demo/archive/dev-environment.tar.gz" >/dev/null
-  $sh_c "tar -zxvf dev.tar.gz --strip-components=1"
+  sh -c "curl -L -o dev.tar.gz https://github.com/eclipse-iofog/demo/archive/v1.3.0-beta.tar.gz" >/dev/null
```

Comment:
Before releasing 1.3.0-rc1, we need to update that URL

Reply:
Agreed, waiting till after we do develop/doc review this week for stuff like that